Source/WebKit/UIProcess/BackingStore.cpp

```diff
@@ -47,8 +47,10 @@ BackingStore::~BackingStore()
 void BackingStore::incorporateUpdate(UpdateInfo&& updateInfo)
 {
     ASSERT(m_size == updateInfo.viewSize);
-    
-    auto bitmap = ShareableBitmap::create(WTFMove(updateInfo.bitmapHandle));
+    if (!updateInfo.bitmapHandle)
+        return;
```

Comment:
@kkinnunen-apple this was the only part I was unsure of. `UpdateInfo` was always designed where `.bitmapHandle` could be `null`. Not sure if `incorporateUpdate` should be called with a null bitmap.

Reply:
https://github.com/WebKit/WebKit/commit/8e8c6b18d4855658cea297e9f8ab27f15592d461 is where `incorporateUpdate` was added and `UpdateInfo.bitmapHandle` was added in https://github.com/WebKit/WebKit/commit/c20f74c0de48c3a58bda5c94203a2cb6a90e991b